spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, :type => :model do
+  it "renames the user" do
+    user =   User.new
+    user.name = 'new_name'
```

Comment:
You can still set custom attributes when creating an object with factory girl, I would check out the documentation